packages/types/src/type/UncleEntryItem.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { BlockNumber } from '.';
+import { Tuple, Option } from '../codec';
```

Comment:
Same here, as above, direct imports. (As done everywhere). Specifically in the RN environment, anything remotely resembling a circular dep is deadly, i.e. it doesn't get past the compilation stages. (JS UIs are much more forgiving here)